FastAPI Assignment 2/test_main.py

```diff
@@ -0,0 +1,42 @@
+from fastapi.testclient import TestClient
+
+from main import app
+
+client = TestClient(app)
+
+
+def test_get_all_songs():
```

Comment:
You need to focus more on how to write unit tests. These are just useless tests.